lib/emscripten/src/utils.rs

```diff
@@ -194,9 +176,7 @@ mod tests {
 
     #[cfg(not(any(feature = "llvm", feature = "clif", feature = "dynasm")))]
     fn get_compiler() -> impl Compiler {
-        panic!("compiler not specified, activate a compiler via features");
-        use wasmer_clif_backend::CraneliftCompiler;
-        CraneliftCompiler::new()
+        panic!("compiler not specified, activate a compiler via features")
```

Comment:
It might be good to use the `compile_error` macro here to prevent people from building a version of wasmer that will fail